.travis.yml

```diff
@@ -22,7 +22,7 @@ before_script:
   - python manage.py collectstatic --noinput
 
 script:
-  - coverage run --source='main' manage.py test --verbosity 2
+  - coverage run --source=main,authentication  manage.py test --verbosity 2
```

Comment:
What happens if you say ```coverage run oshc/manage.py test```?

Reply:
Then our coverage decreased to 42% because Travis makes virtual environment and `test.py` only covers some file